src/Config.ts

```diff
@@ -13,6 +13,11 @@ export interface Config {
      */
     type?: string;
 
+    /**
+     * Array of type names to generate schemas for. Cannot be used with --type.
+     */
+    types?: string[];
```

Comment:
can't we just allow the user to pass multiple `--type` parameters?

`--type A --type B` instead of `--types` ?

Reply:
Good point. What's more common in clis?